common/web/lm-worker/build.sh

```diff
@@ -95,12 +95,16 @@ fi
 # of typescript, we need to avoid this!
 # TODO: we should try and rework this to avoid the need to manually wrap
 
-if builder_start_action clean || builder_has_action build; then
+if builder_start_action clean; then
   npm run clean
   builder_finish_action success clean
 fi
 
 if builder_start_action build; then
+  if ! builder_has_action clean; then
+    npm run clean
+  fi
```

Comment:
Changes this:

![image](https://user-images.githubusercontent.com/25213402/191646450-b62f88b0-937c-4353-b809-11505cd34665.png)

(Note the warning in yellow.)

to this:

![image](https://user-images.githubusercontent.com/25213402/191647177-a5da3c5d-d6da-437e-b021-6d1bce744746.png)

Note how the `npm run clean` is run _after_ the `build:project starting` log message, as part of that action - but only if the `clean` action is not specified.